implementation/src/main/java/io/smallrye/config/DirConfigSource.java

```diff
@@ -16,23 +16,28 @@
 
 package io.smallrye.config;
 
+import org.eclipse.microprofile.config.spi.ConfigSource;
+import org.jboss.logging.Logger;
```

Comment:
Was a local build run with these changes, as I don't think the imports should be reordered in the PR?